docs/ENVIRONMENT.md

```diff
@@ -632,6 +632,14 @@ SYSTEMD_HOME_DEBUG_SUFFIX=foo \
   file (containing firmware measurement data) to read. This allows overriding
   the default of `/sys/kernel/security/tpm0/binary_bios_measurements`.
 
+`systemd-sleep`:
+
+* `$SYSTEMD_SLEEP_SKIP_FREEZE_USER_SESSIONS` - Takes a boolean. When true,
+  `user.slice` will not be frozen during sleep. We recommend against using this
+  variable, because it can lead to unexpected behavior. This is especially true
```

Comment:
that's a bit vague

Reply:
i mean, vague is fine, but maybe not thaaaat vague.

maybe say that applications might not expect their home dir being frozen and simiöar

Reply:
Better? I put the detailed notes under the homed env var (since IMO that's the more dangerous one to toggle) and the sleep one is left somewhat vague because, well, it is a somewhat vague issue